samples/unit-tests/series/datagrouping-anchor/demo.js

```diff
@@ -0,0 +1,273 @@
+QUnit.test('Data grouping anchor for points in the middle of the data set.', function (assert) {
+    const chart = Highcharts.stockChart('container', {
+            yAxis: [{
+                height: '33,33%',
+                offset: 0
+            }, {
+                height: '33,33%',
+                top: '33,33%',
+                offset: 0
+            }, {
+                height: '33,33%',
+                top: '66,66%',
+                offset: 0
+            }],
+            plotOptions: {
+                series: {
+                    data: [1, 2, 3, 4, 1, 2, 3, 4, 1, 2, 3, 4],
+                    pointInterval: 3600 * 1000,
+                    dataGrouping: {
+                        approximation: 'average',
+                        enabled: true,
+                        forced: true,
+                        units: [
+                            ['hour', [2]]
+                        ]
+                    }
+                }
+            },
+            series: [{
+                yAxis: 0
+            }, {
+                yAxis: 1,
+                dataGrouping: {
+                    anchor: 'middle'
+                }
+            }, {
+                yAxis: 2,
+                dataGrouping: {
+                    anchor: 'end'
+                }
+            }]
+        }),
+        hour = 3600 * 1000;
+
+    assert.strictEqual(
+        chart.series[0].points[1].x,
```

Comment:
Question: Why checking the second point?

Reply:
My intention was to test points that are not extremes that's why I chose the point with the first index (second point).